cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SpatialTest.scala

```diff
@@ -52,7 +54,7 @@ class SpatialTest extends DocumentingTest {
           |* Each point will also be associated with a specific <<cypher-spatial-crs,Coordinate Reference System>> (CRS) that determines the meaning of the values in the _Coordinate_.
           |* Instances of _Point_ and lists of _Point_ can be assigned to node and relationship properties.
           |* Nodes with _Point_ or _List(Point)_ properties can be indexed using a spatial index. This is true for all CRS (and for both 2D and 3D).
-          |  There is no special syntax for creating spatial indexes, as it is supported using the existing <<administration-indexes-create-a-single-property-index,indexes>>.
+          |  There is no special syntax for creating spatial indexes, as it is supported using the existing <<administration-indexes-create-a-single-property-index-for-nodes,indexes>>.
```

Comment:
```suggestion
          | * There is no special syntax for creating spatial indexes, as it is supported using the existing <<administration-indexes-create-a-single-property-index-for-nodes,indexes>>.
```

Reply:
Right now this is not its own bullet point, but part of the bullet point above.

Reply:
Not my area, I just updated the link :shrug: not sure if it is supposed to be its own point or not

Reply:
Since the point above also is about indexes, this might be intentional actually. Just thought it looked a bit odd